data/partners.js

```diff
@@ -0,0 +1,68 @@
+import celo from "/public/assets/images/logo-celo.webp";
```

Comment:
importing the images from the public dir can result into a big production bundle. may be we can suggest you pass the path only like `/images/logo-symplifi.png`